docs/Outrounds.md

```diff
@@ -0,0 +1,27 @@
+Outrounds
+========
+
+Outrounds are now here!
+
+Handling the Break
+------------------
+
+When you reach the final in-round of your tournamet, the button that is normally "Prepare Next Round" should read "Break 'em".  If this does not happen, please consult your `tot_rounds` TabSetting and ensure it is set properly.
+
+This will then break the appropriate number of teams as determined by the `nov_teams_to_break` and `var_teams_to_break` TabSetting.
+
+It will also perform a number of checks to ensure you have enough rooms and judges.  It does support paneled rounds (as long as they are consistently paneled) -- please consult the `nov_panel_size` and `var_panel_setting` for more information.
```

Comment:
Not sure that we need to validate the panel size:

1. I think we can trust most tournaments to do this themselves

2. Sometimes panels end up being inconsistently-sized due to judging constraints and I don't want to make tournaments have to struggle with the software in that instance (For example: Dartmouth my Freshman year had a 1-person panel in novice finals)

Reply:
I think validating it for the like about to pair screen is still helpful.  In status quo it still lets you pair even if you don’t have enough judges so not sure what change you are suggesting — perhaps it should be more explicit that for judges it is a warning that they can safely ignore, though I still think it could help people do the math quickly.

Reply:
In new commit, I took this validation out of the `good_to_go` method, but it remains in the UI.